demos/general/demo_pipeline_voltage_imaging.py

```diff
@@ -172,9 +192,10 @@ def main():
     min_spikes= 10                                # minimal spikes to be found
     threshold = 3.5                               # threshold for finding spikes, increase threshold to find less spikes
     do_plot = False                               # plot detail of spikes, template for the last iteration
-    ridge_bg= 0.001                               # ridge regression regularizer strength for background removement
+    ridge_bg= 0.5                               # ridge regression regularizer strength for background removement
```

Comment:
How would someone know from the comment what different values of this mean?

Reply:
I added something